README.md

```diff
@@ -108,7 +108,7 @@ Draftail, like Draft.js, distinguishes between 3 content formats:
 Common formatting options are available out of the box:
 
 - Block types: `H1`, `H2`, `H3`, `H4`, `H5`, `H6`, `Blockquote`, `Code`, `UL`, `OL`, `P`
-- Inline styles: `Bold`, `Italic`, `Underline`, `Monospace`, `Strikethrough`
+- Inline styles: `Bold`, `Italic`, `Code`, `Underline`, `Strikethrough`, `Mark`, `Keyboard`, `Superscript`, `Subscript`
```

Comment:
There are more, but I chose to leave them out because they are seldom used in rich text editors.